services/meta/renameFile_unix.go

```diff
@@ -0,0 +1,12 @@
+// +build !windows
+
+package meta
+
+import (
```

Comment:
Please use [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports) for formatting files. `goimports` produces this output for only a single import statement.

```
import "os"
```